core/__pypackages__/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
I don't think we should include this folder here

Is there a specific reason for this?

Reply:
Nope, that's a mistake.